web/src/components/data-entities/site/SiteList.js

```diff
@@ -10,152 +10,173 @@ import {entityDelete} from '../../../api/api';
 import 'ag-grid-enterprise';
 
 const SiteList = () => {
-  const [gridApi, setGridApi] = useState();
-  const [redirect, setRedirect] = useState();
-  const [dialogState, setDialogState] = useState({open: false});
+    const [redirect, setRedirect] = useState();
```

Comment:
Tab size should be set to 2. Looks like it's set to 4?

Reply:
Updated and removed dependencies